edit/usw-linking.js

```diff
@@ -0,0 +1,57 @@
+/* global $ $create $remove */// dom.js
+/* global editor */
+
+'use strict';
+
+let uswPort;
+
+function connectToPort() {
+  if (!uswPort) {
+    uswPort = chrome.runtime.connect({name: 'link-style-usw'});
+    uswPort.onDisconnect.addListener(err => {
+      throw err;
+    });
+  }
+}
+
+/* exported linkToUSW */
+function linkToUSW() {
+  connectToPort();
+
+  uswPort.postMessage({reason: 'link', data: editor.style});
+}
+
+/* exported revokeLinking */
+function revokeLinking() {
+  connectToPort();
+
+  uswPort.postMessage({reason: 'revoke', data: editor.style});
+}
+
+/* exported uploadStyle */
+function uploadStyle() {
+  connectToPort();
+  const data = Object.assign(editor.style, {sourceCode: editor.getEditors()[0].getValue()});
```

Comment:
I didn't find anything related to textcontent on the style's object, so doing this little hack should do it.